packages/abi-gen-templates/partials/tx.handlebars

```diff
@@ -27,6 +33,13 @@ public {{languageSpecificName}} = {
     const txHash = await self._web3Wrapper.sendTransactionAsync(txDataWithDefaults);
     return txHash;
     },
+    /**
+     * Sends the transaction and wait for it to succeed
```

Comment:
`Sends an Ethereum transaction and waits until the transaction has been successfully mined without reverting. If the transaction was mined, but reverted, an error is thrown.`